### PR TITLE
CXSPA-6816: Fix schematics.

### PR DESCRIPTION
**Pull request by tiwariakshay55**

### CODE REVIEW

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap [hunk 1]
```diff
@@ -1,134 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature eager loading should import appropriate modules 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature eager loading should import appropriate modules 1`] = `""`;
```
Comment (kpawelczak) [inline feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap]: Could you please help me understand why there is no output for this test?

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap [hunk 2]
```diff
@@ -1,134 +1,21 @@
...
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
```
Comment (kpawelczak) [inline feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:3]: Same situation as above

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap [hunk 3]
```diff
@@ -1,134 +1,21 @@
...
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup styling should create a proper scss file 1`] = `
-"@import "../../styles-config";
-@import "@spartacus/estimated-delivery-date";
-"
-`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 2`] = `
```
Comment (kpawelczak) [inline feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:5]: I'm not sure if there's a mistake in the text or the output. It mentions that it should be loaded using lazy loading syntax, but I don't see it.

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap [hunk 4]
```diff
@@ -1,134 +1,21 @@
...
+"import { NgModule } from '@angular/core';
+import { CartBaseModule } from "@spartacus/cart/base";
+import { EstimatedDeliveryDateModule } from "@spartacus/estimated-delivery-date";
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup styling should update angular.json 1`] = `
```
Comment (kpawelczak) [inline feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:13]: Why was this test removed? does this feature use `estimated-delivery-date.scss`?

**Review by kpawelczak — CHANGES_REQUESTED** [review]

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap (hunk 4 above)
Comment (tiwariakshay55) [reply feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:13]: Removed by mistake added it back

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap (hunk 3 above)
Comment (tiwariakshay55) [reply feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:5]: we are lazy loading estimated delivery date via cart we have introduced a wrapper for the same

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap (hunk 1 above)
Comment (tiwariakshay55) [reply feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap]: Opening this as a bug in the ticket and checking this.

File: feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap (hunk 2 above)
Comment (tiwariakshay55) [reply feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap:3]: Opening this as a bug in the ticket and checking this.

**Review by kpawelczak — APPROVED** [review]

**Pull request merged**